core/src/main/java/io/undertow/server/protocol/http/HttpRequestParser.java

```diff
@@ -355,6 +361,13 @@ private void handleStateful(ByteBuffer buffer, ParseState currentState, HttpServ
         }
     }
 
+    private void sanitazeListTypeHeaders(final HttpServerExchange builder) {
+        //NOTE: this should be used only for HTTP1.x
+        if(!Cookies.isCrumbsAssemplyDisabled()) {
+            Cookies.assembleCrumbs(builder.getRequestHeaders());
+        }
+        //TODO: list type headers?
```

Comment:
Wdy mean here?

Reply:
Supposedly there are list type headers - CSV style, though I wasnt able to find any existing example. If such headers are present( multiple instances) those should be squashed into single instance with CSV content.